### PR TITLE
bump sdk to latest

### PR DESCRIPTION
**Pull request by piyalbasu**
To keep parity with freighter-backend, upgrading stellar-sdk to the latest.

This doesn't appear to have any effect on the app, but is just to keep the versions tidy and in the event that our tests don't cover some edge case that could be resolved by this latest version

### CODE REVIEW

File: extension/src/popup/views/Account/hooks/useGetAccountHistoryData.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import { useReducer } from "react";
```
Comment (aristidesstaffieri) [inline extension/src/popup/views/Account/hooks/useGetAccountHistoryData.tsx]: is this intended?
